### PR TITLE
DSD-1211: visited state for Link component

### PR DESCRIPTION
**Pull request by oliviawongnyc**
Fixes JIRA ticket [DSD-1211](https://jira.nypl.org/browse/DSD-1211).

## This PR does the following:

- Adds `ui.link.tertiary` and `dark.ui.link.tertiary` to the color pallete.
- Adds `visited` state to the `Link` component for both `light mode` and `dark mode`.

## How has this been tested?

- On Storybook

## Accessibility concerns or updates

- No, this is an accessibility improvement

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/theme/foundations/colors.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ const colors: Colors = {
       primary: linkPrimary,
       "primary-05": hexToRGB(linkPrimary, 0.05),
       secondary: linkSecondary,
+      tertiary: "#551A8B",
```
Comment (EdwinGuzman) [inline src/theme/foundations/colors.ts]: Create a variable for this after `linkPrimary` and `linkSecondary` on line 37.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
The update itself looks good (though I'm pro "underlined links all the time"). Update the version in the `Link`'s storybook doc and also update its Accessibility section. @bigfishdesign13 maybe we can get some text from Clare for the removal of the underline for links. For visited links, what's in the ticket is a good start, "For accessibility and UX purposes, a visited state should be added to links."

**Comment by oliviawongnyc** [comment]
> The update itself looks good (though I'm pro "underlined links all the time"). Update the version in the `Link`'s storybook doc and also update its Accessibility section. @bigfishdesign13 maybe we can get some text from Clare for the removal of the underline for links. For visited links, what's in the ticket is a good start, "For accessibility and UX purposes, a visited state should be added to links."

Added a little line about visited states. I don't know that all of the button variants have visited states.

**Review by EdwinGuzman — COMMENTED** [review]
I'm good with these changes considering more a11y documentation will be added later (as a separate ticket?).

**Comment by oliviawongnyc** [comment]
> I'm good with these changes considering more a11y documentation will be added later (as a separate ticket?).

Or should we wait for Clare's input @bigfishdesign13?

**Comment by bigfishdesign13** [comment]
> > I'm good with these changes considering more a11y documentation will be added later (as a separate ticket?).
> 
> Or should we wait for Clare's input @bigfishdesign13?

@oliviawongnyc 
We can mark this PR as `approved` and then create a seprate ticket fot the docs updates.  That said, this should not be merged yet.

**Review by EdwinGuzman — APPROVED** [review]
Don't merge yet.

**Comment by oliviawongnyc** [comment]
@bigfishdesign13, @EdwinGuzman – removed any changes related to underline from this PR

**Review by EdwinGuzman — APPROVED** [review]

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.

**Pull request merged**